src/server/api/routers/smartsheet/sheets/index.ts

```diff
@@ -0,0 +1,21 @@
+import { z } from "zod";
+import type { Sheet } from "../smartsheetTypes";
+import { createTRPCRouter, publicProcedure } from "../../../trpc";
+
+const sheetRouter = createTRPCRouter({
+  getSheets: publicProcedure.query(({ ctx }) => {
+    const options = {
+      queryParameters: {
+        include: "attachments",
+        includeAll: true,
+      },
+    };
+    const res: Sheet[] = ctx.smartsheet.sheets.listSheets(options);
+    return res;
+  }),
+  getSheet: publicProcedure.input(z.number()).query(({ ctx, input }) => {
```

Comment:
I tend to prefer to avoid naming routes with only one letter difference, maybe call this getUniqueSheet or something if you don't mind. Makes it less easy to use the wrong one or misunderstand something when reading the code.

Reply:
I got rid of getSheets since I won't need it